### DISCUSSION TITLE
loco position node led sequence powered

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1084] [Q&A]
Hi, The mirco usb of the anchor has damaged so I replace to  the barrel jack to provide power. But just like the picture show the light is one blue and one red. What does it means ?
![Uploading IMG_3488.jpeg…]()

**Comment by knmcguire** [comment]
Hi! I think something went wrong with uploading your picture. Could you try again?

**Reply by CalvinKoung** [reply]
ok, sorry
![IMG_3488](https://github.com/bitcraze/discussions/assets/118436188/f91f9e3a-b5b9-4cac-91ff-ba8bcaf97c35)

**Comment by knmcguire** [comment]
Hmm... so only the power and ranging LED are on... I can not find anywhere in the code or the LED explanation (tab specifications on the [loco node product page](https://www.bitcraze.io/products/loco-positioning-node/)) what this means so I can only speculate that the chip somehow freezes. This might be because of the micro usb port being broken, but perhaps it can also be that the anchor is [not properly powered.](https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/#powering-the-anchors) . 

Are you sure the jack is providing between 5-12 V and at least 150 mA of current?

**Reply by CalvinKoung** [reply]
ok, so weird. After try the more times, sometimes it can run normally but sometimes will apear the above condition.

**Reply by knmcguire** [reply]
that seems to me as a power issue. Could you check the specs of the power adapter used for the barrel jack? Perhaps you can check it yourself if you have a multimeter?
